lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -130,6 +130,7 @@ def stop_repmgrd
 
     def node_number_valid?
       rec = record_for_node_number
+      return false if rec == false
```

Comment:
This seems a bit odd especially because we're returning `true` if rec is `nil`. I think we're overloading the return value of `record_for_node_number` too much here.

Maybe it would be better to rescue the exception in this method. What do you think?